README.md

```diff
@@ -140,6 +140,45 @@ For a fair comparison, [LogLevel](http://npm.im/loglevel) was extended
 to include a timestamp and [bole](http://npm.im/bole) had
 `fastTime` mode switched on.
 
+<a name="transports"></a>
+## Transports
+
+A transport in most logging libraries is something that runs in-process to
+perform some operation with the finalized log line. For example, a transport
+might send the log line to a standard syslog server after processing the log
+line and reformatting it. For details on implementing, and some already written,
+transports, see our [Transports⇗](docs/transports.md) document.
+
+> **Pino *does not* natively support in-process transports.**
+
+Pino does not support in-process transports because Node processes are
+single threaded processes (ignoring some technical details). In order for Pino
+to achieve the speed it achieves, Pino purposefully offloads the handling of
```

Comment:
Hmm, I meant to imply that it is one of the factors in Pino's speed. How about "...details). Given this restriction, one of the methods Pino employs to achieve its speed is to purposfully offload..."?